main/lp/lp_subscribe_users.php

```diff
@@ -159,7 +159,7 @@
 }
 $form->setDefaults($defaults);
 
-$currentUser = api_get_user_entity(api_get_user_id());
+$currentUser = api_get_user_entity();
```

Comment:
`api_get_user_entity` always needs an argument
https://github.com/chamilo/chamilo-lms/blob/1.11.x/main/inc/lib/api.lib.php#L1798

Reply:
This pull request also changes api_get_user_entity definition: the argument becomes optional - in cas it is not provided, the current logged-in user id is automatically substituted. As it is already the case with the other api_get_*_entity global functions : api_get_course_entity() returns the current course, api_get_session_entity() returns the current session, and so on.

Reply:
I should have splitted this too big PR in a sequence of smaller ones.

Reply:
> This pull request also changes api_get_user_entity definition: the argument becomes optional - in cas it is not provided, the current logged-in user id is automatically substituted. As it is already the case with the other api_get_*_entity global functions : api_get_course_entity() returns the current course, api_get_session_entity() returns the current session, and so on.

Ok. The diff is very long for api.lib.php and the file is not shown